apps/public-docsite-resources/package.json

```diff
@@ -1,10 +1,19 @@
 {
   "name": "@fluentui/public-docsite-resources",
   "version": "1.0.0",
-  "private": true,
   "description": "Fluent UI React local demo app",
   "main": "lib-commonjs/index.js",
   "module": "lib/index.js",
+  "publishConfig": {
```

Comment:
Do we need this everywhere? Currently we have it in some packages but not others.